### ISSUE TITLE
[Feature Request] Per table theming

### ISSUE THREAD

**Issue by bxschmidt** [Kreyu/data-table-bundle#20]
Great project :+1:

Would be nice if there was a way to do something like

{{ data_table(table, {'theme': 'data_table/theme_a.html.twig'}) }}
{{ data_table(table, {'theme': 'data_table/theme_b.html.twig'}) }}

to not only have the theme be determined by bundle configuration but to be able to determine theme everytime a table is rendered.

Feature request in case i am not able to create a pull request with the feature.
Best regards

**Comment by Kreyu** [comment]
Hey, thanks for suggestion! I agree, this seems like a _must have_ feature and I'll try to implement this as soon as possible.

**Comment by Kreyu** [comment]
Per table theming is now available since version 0.12.0.
The configuration `themes` node is now deprecated, use `defaults.themes` instead.

Of course, you can override the `themes` option directly in the template (notice the "theme**s**" array instead of singular "theme"):

```
{{ data_table(table, {'themes': ['data_table/theme_a.html.twig']}) }}
```

The documentation is also updated: 
- https://data-table-bundle.swroblewski.pl/features/theming/
- https://data-table-bundle.swroblewski.pl/reference/configuration/
